NOnion/StreamState.fs

```diff
@@ -0,0 +1,11 @@
+﻿namespace NOnion
+
+open System.Threading.Tasks
+
+type StreamState =
+    | Initialized
+    | Connecting of
+        streamId: uint16 *
+        completionTcs: TaskCompletionSource<uint16>
```

Comment:
What's Tcs? seems this could use a better name

Reply:
TaskCompletionSource

Reply:
so "completionTaskCompletionSource"? see what I mean?

Reply:
^

Reply:
Fixed